app/templates/views/join-service-request-approver.html

```diff
@@ -0,0 +1,24 @@
+{% extends "withnav_template.html" %}
+{% from "components/page-header.html" import page_header %}
+{% from "components/page-footer.html" import page_footer %}
+{% from "components/form.html" import form_wrapper %}
+
+{% block service_page_title %}
+  {{ '{} wants to join your service'.format(requester.name) }}
+{% endblock %}
+
+{% block maincolumn_content %}
+
+  {{ page_header('{} wants to join your service'.format(requester.name)) }}
+
+  {% call form_wrapper() %}
+    <p class="bottom-gutter">
+      {{ "Do you want to let {} join '{}'?".format(requester.name, requested_service.name)}}
```

Comment:
Change to ‘smart’ (curly) quotes.

```suggestion
      {{ "Do you want to let {} join ‘{}’?".format(requester.name, requested_service.name)}}
```